docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,485 @@
-# Man Page
+# Man pages
+
+## Commands by functional group
+
+### Performance tool management commands
+
+- [pbench-clear-results](#pbench-clear-results)
+- [pbench-clear-tools](#pbench-clear-tools)
+- [pbench-list-tools](#pbench-list-tools)
+- [pbench-list-triggers](#pbench-list-triggers)
+- [pbench-register-tool](#pbench-register-tool)
+- [pbench-register-tool-set](#pbench-register-tool-set)
+- [pbench-register-tool-trigger](#pbench-register-tool-trigger)
+
+### Benchmark commands
+
+- [pbench-user-benchmark](#pbench-user-benchmark)
+
+### Upload to Pbench Server
+
+#### Pbench Server 0.69
+
+- [pbench-move-results](#pbench-move-results)
+- [pbench-copy-results](#pbench-copy-results)
+
+#### Pbench Server 1.0
+
+- [pbench-results-move](pbench-results-move)
+
+## Commands
+
+---
+
+### pbench-clear-results
+
+---
+
+**NAME**  
+`pbench-clear-results` - clears the result directory
+
+**SYNOPSIS**  
+`pbench-clear-results [OPTIONS]`
+
+**DESCRIPTION**  
+This command clears the results directory from `/var/lib/pbench-agent` directory.
+
+**OPTIONS**  
+ `-C`, `--config` PATH  
+Path to the Pbench Agent configuration file.
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+
+`--help`  
+Show this message and exit.
+
+---
+
+### pbench-clear-tools
+
+---
+
+**NAME**  
+`pbench-clear-tools` - clear registered tools by name or group
+
+**SYNOPSIS**  
+`pbench-clear-tools [OPTIONS]`
+
+**DESCRIPTION**  
+Clear all tools which are registered and can filter by name of the group.
+
+**OPTIONS**  
+`-C`, `--config` PATH  
+Path to the Pbench Agent configuration file.
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+
+`-n`, `--name`, `--names` STR  
+Clear only the <`STR`> tool.
+
+`-g`, `--group`, `--groups` STR  
+Clear the tools in the <`STR`> group. If no group is specified, the `default` group is assumed.
+
+`-r`, `--remote`, `--remotes` STR[,STR...]  
+Clear the tool(s) only on the specified remote(s). Multiple remotes may be specified as a comma-separated list. If no remote is specified, all remotes are cleared.
+
+`--help`  
+Show this message and exit.
+
+---
+
+### pbench-copy-results
+
+---
+
+**NAME**  
+`pbench-copy-results` - copy result tarball to the 0.69 Pbench Server
+
+**SYNOPSIS**  
+`pbench-copy-results [OPTIONS]`
+
+**DESCRIPTION**  
+Push the benchmark result to the Pbench Server without removing it from the local host. This command requires `/opt/pbench-agent/id_rsa` file with the private SSH key, when pushing to a 0.69 Pbench Server.
+
+**OPTIONS**  
+`--user`  
```

Comment:
`--user STR` or (better, although inconsistent with the other examples), `--user <user>`.